tests/framework/e2e/cluster.go

```diff
@@ -182,6 +183,8 @@ type EtcdProcessClusterConfig struct {
 	CorruptCheckTime        time.Duration
 	CompactHashCheckEnabled bool
 	CompactHashCheckTime    time.Duration
+
+	ClientAuthOpts []config.ClientOption
```

Comment:
I don't think we need this.

Reply:
We need to make epc.StartNewProc() work correctly in auth enabled mode.

Reply:
I suggest to add parameter `opts ...config.ClientOption` for both `StartNewProc` and `CloseProc`.